test/james.testing.clss.pas

```diff
@@ -0,0 +1,53 @@
+unit james.testing.clss;
+
+{$include james.inc}
+
+interface
+
+uses
+    james.testing,
+  {$IFDEF FPC}
+    fpcunit, james.testing.clss.lazarus
+  {$ELSE}
+    TestFramework, james.testing.clss.delphi
+  {$ENDIF}
+  ;
+
+type
+  {$IFDEF FPC}
+    TTestCase = FPCUnit.TTestCase;
+  {$ELSE}
+    TTestCase = TestFramework.TTestCase;
+  {$ENDIF}
```

Comment:
The syntax would have to be `<T: TTestCase>`.
I could not get it to work on FPC though. Seems not to be implemented at this point, neither in FPC or Delphi.

In Delphi, it is possible to simulate an alias through inheritance, like this:
`TTest<T: TTestCase> = class (James.Testing.Delphi.TTest<T>);`
(removing the **sealed** keyword from the TTest class)

But this also doesn't work in FPC.